specs/opcode/3eRETURNDATACOPY.md

```diff
@@ -0,0 +1,55 @@
+# RETURNDATACOPY opcode
+
+## Procedure
```

Comment:
Any special behavior to document when `is_root`, etc... ?

Reply:
I think is_root has nothing to do with the returndatacopy, as it depends only on the previous call. If there is no previous call (no matter current call is root or not), returndatacopy gets 0 as its boundary (evm checks returndatasize which is 0 here), if param length is also 0, everything is still ok, if length > 0, then the context reverts as expected.